### DISCUSSION TITLE
Ask for some recommend methods.

### DISCUSSION THREAD

**Discussion by StefanIsSmart** [pyg-team/pytorch_geometric#8762] [General]
If I have a graph, that is N nodes and N * N edges (Nodes and Edges all have many features), Is there any graph NN recommends using?
As we all know, the graph is very easy to over-smoothing. (Especially fully connected NetWork.)
I also need a GNN that could catch both the Node and Edge features to get an embedding of them.
Is there some recommended model?

Thanks!

**Comment by rusty1s** [comment]
For fully-connceted networks, you will need some anisotropic GNN, e.g., using attention, or something like `EdgeConv`.
